dotcom-rendering/src/types/tagFront.ts

```diff
@@ -23,8 +23,44 @@ export interface FETagFrontType {
 	forceDay: boolean;
 }
 
+/**
+ * Represents a set of trails grouped by their year, month & optionally day of publication.
+ */
+export interface GroupedTrailsBase {
+	year: number;
+	month: number;
+	day: number | undefined;
+}
+
+export interface GroupedTrails extends GroupedTrailsBase {
+	trails: DCRFrontCard[];
+}
+
+export interface GroupedTrailsFastMpu extends GroupedTrailsBase {
+	injected: true;
+	speed: 'fast';
+	// Trails must either be length of 2, 4, 6 or >= 9
+	trails:
+		| Tuple<DCRFrontCard, 2>
+		| Tuple<DCRFrontCard, 4>
+		| Tuple<DCRFrontCard, 6>
+		| [...Tuple<DCRFrontCard, 9>, ...DCRFrontCard[]];
```

Comment:
Could this become `Tuple<DCRFrontCard, 9 | Infinity>`?

Reply:
Tuple only supports up to 12 right, so not entirely sure what this would output? I like how explicit the current definition feels - you can clearly see what you're getting!